counsel.el

```diff
@@ -2424,6 +2424,11 @@ FZF-PROMPT, if non-nil, is passed as `ivy-read' prompt argument."
                         (message (cdr x)))
               :caller 'counsel-rpm)))
 
+(defcustom counsel-file-jump-args " * -type f -not -path '*\/.git*'"
```

Comment:
I don't like to think about that sneaky leading space when customizing.  The `counsel-file-jump` should be smart enough to add one if necessary.